beetsplug/playlist.py

```diff
@@ -65,7 +66,7 @@ def __init__(self, pattern):
             f.close()
             break
 
-    def col_clause(self):
+    def clause(self) -> Tuple[Optional[str], Sequence[Any]]:
```

Comment:
Wow, very nice catch. That's the cost of having "transparent" fallback for queries with no SQL implementation; you don't notice when the SQL implementation is just silently not being used.